src/content/en/2019/third-parties.md

```diff
@@ -0,0 +1,167 @@
+---
+part_number: II
+chapter_number: 5
+title: Third Parties
+authors: [patrickhulce]
+reviewers: [simonhearne, flowlabs, jasti, zeman]
+translators: []
+---
+
+# Chapter 5: Third Parties
+
+## Introduction
+
+The open web is vast, linkable, and interoperable by design. The ability to grab someone else’s complex library and use it on your site with a single `<link>` or `<script>` element has supercharged developers’ productivity and enabled awesome new web experiences; on the flip side, the immense popularity of a select few third-party providers raises important performance and privacy concerns. This chapter examines the prevalence and impact of third-party code on the web in 2019, the web usage patterns that lead to the popularity of third-party solutions, and potential repercussions for the future of web performance and privacy.
+
+## Definitions
+
+### “Third Party”
+
+A third party is an entity outside the primary site-user relationship, i.e. the aspects of the site not directly within the control of the site owner but present with their approval.
```

Comment:
Adding a quick "For example, \<XYZ\> library" would help make sure every reader is on the same page here.

Reply:
great idea! 👍